app/services/pokemon_index_service.rb

```diff
@@ -0,0 +1,24 @@
+include Rails.application.routes.url_helpers
+
+class PokemonIndexService
```

Comment:
in an ideal world the services would also be tested, but given that this service only has a single method that ends up being tested by the API testing, its ok.